### PR TITLE
Konstantinos / [FE Deriv Squad] [Office] Berlin page in [Our Locations] 

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [x] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/careers/_model/_locations/_locations.ts [hunk 1]
```diff
@@ -375,6 +376,28 @@ export const guernsey: LocationsType = {
...
+    img_alt: 'berlin alt img',
+    map_office_name: '(Germany)',
+    country: 'Germany',
+    flagIcon: GermanyFlagIcon,
+    thumbnail: 'thumbnail_berlin',
+    address: address_guernsey,
+    first_p:
+        'Deriv (Germany) will open later this year in the heart of the European Union. Home to tech professionals, this agile office will help the Deriv Group with its most ambitious technological challenges to create future-proof online trading solutions. ',
+    first_img: 'living_in_berlin',
+    first_img_alt: 'living_in_berlin alt img',
```
Comment (habib-deriv) [inline src/pages/careers/_model/_locations/_locations.ts:394]: Image all issue

File: crowdin/messages.json [hunk 2]
```diff
@@ -1832,6 +1833,7 @@
   "-197830563": "The ability to access a limited portion of clients’ sensitive information, our back-end code, or internal information on GitHub",
   "-611936805": "Exposed logs without sensitive information",
   "-1076255913": "Asunción | Our office – Paraguay | Deriv",
+  "-241792402": "Berlin – Our office | Deriv",
```
Comment (hamid-deriv) [inline crowdin/messages.json:1836]:
```suggestion
  "-241792402": "Berlin | Our office | Deriv",
```

Comment (hamid-deriv) [reply crowdin/messages.json:1836]: I don't have access to the content file, I just suggest this based on the other translations.

File: src/pages/careers/locations/berlin/index.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    }
+`
+
+const Berlin = () => {
+    const images = useStaticQuery(query)
+
+    return (
+        <Layout type="careers">
+            <SEO
+                title={localize('Berlin – Our office | Deriv')}
```
Comment (hamid-deriv) [inline src/pages/careers/locations/berlin/index.tsx:41]:
```suggestion
                title={localize('Berlin | Our office | Deriv')}
```

File: crowdin/messages.json (hunk 2 above)
Comment (konstantinosG-deriv) [reply crowdin/messages.json:1836]: https://docs.google.com/document/d/16Nq-k26sUzovcjRyyej-pfxdcvgInQJ6NvQQxgkTM_o/edit# 
That's the file I followed, should I keep it as it is?

Comment (hamid-deriv) [reply crowdin/messages.json:1836]: Actually, I don't have access y to google docs.

File: src/pages/careers/locations/berlin/index.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    }
+`
+
+const Berlin = () => {
+    const images = useStaticQuery(query)
+
+    return (
+        <Layout type="careers">
+            <SEO
+                title={localize('Berlin | Our office | Deriv')}
```
Comment (habib-deriv) [inline src/pages/careers/locations/berlin/index.tsx:41]:
```suggestion
                title={localize('Berlin - Our office | Deriv')}
```

**Review by habib-deriv — DISMISSED** [review]
LGTM

**Review by habib-deriv — APPROVED** [review]
LGTM

**Pull request merged**